packages/govuk-frontend-review/package.json

```diff
@@ -34,7 +35,6 @@
     "marked": "^5.1.1",
     "nunjucks": "^3.2.4",
     "outdent": "^0.8.0",
-    "prettier": "^2.8.8",
```

Comment:
I choose use CommonJS module as entry for two reasons, 

1. prettier v3 still support `require()` to load module, if I export `@prettier/sync` as esm-only module, people may complain. (I don't want introduce a bundler)
2. `format` is a function, it can be named-exported, but `version` is a getter, it can't be named-exported.

So `@prettier/sync` mean to only support default export.

Maybe we can make named export work with [this trick](https://github.com/evanw/esbuild/issues/2874#issue-1561245328), but I don't think worth to make things strange.

Reply:
Yeah definitely, in that case we should update the types to show this is invalid:

```mjs
import { format } from '@prettier/sync'
```

I'll open another PR